models/DataObject/Concrete.php

```diff
@@ -574,15 +574,15 @@ public function getNextParentForInheritance()
      */
```

Comment:
AbstractObject or Concrete ?

Reply:
It was `AbstractObject` before as well. 
But yes, in my opinion  `AbstractObject` is right, because we if the direct parent of the called object can be a folder, and that should definitely work, right?